modules/fbs-core/web/src/app/tool-components/math-input/math-input.component.scss

```diff
@@ -3,6 +3,10 @@ math-field {
   font-size: large;
 }
 
+:host ::ng-deep .ML__content {
+  overflow: unset;
+}
+
```

Comment:
```suggestion
```

It looks like this code doesn't do anything/anything anymore? 
Since otherwise the changes in [`math-input.component.ts`](https://github.com/thm-mni-ii/feedbacksystem/pull/1401/files#diff-361fbf020a5ef6ec87cf5ef8bce5023a82f8bccf452d1aedd894c7dedd63d22e) (see below) would be superfluous?

https://github.com/thm-mni-ii/feedbacksystem/blob/77be029535db8a79177878303d67ef8333a35af2/modules/fbs-core/web/src/app/tool-components/math-input/math-input.component.ts#L65-L69